submissions/team_api.py

```diff
@@ -192,6 +191,40 @@ def _log_team_submission(team_submission_data):
     )
 
 
+def get_submissions_from_other_teams(
+    team_member_ids,
```

Comment:
this is like, the tiniest nit, but could you change the order of params to course, item, team? it's how we order it pretty mush everywhere else in this file, since it's sorta in descending """size""" order